jobs/JRRFS_UNGRIB

```diff
@@ -3,13 +3,20 @@ declare -rx PS4='+ $(basename ${BASH_SOURCE[0]:-${FUNCNAME[0]:-"Unknown"}})[${LI
 set -x
 date
 #
-# source the config cascade
-source ${EXPDIR}/exp.setup
-[[ ! -z "${ENS_INDEX}" ]] && source ${EXPDIR}/config/config.ens
-source ${EXPDIR}/config/config.${MACHINE}
-source ${EXPDIR}/config/config.base
-[[ -s ${EXPDIR}/config/config.ungrib ]] && source ${EXPDIR}/config/config.ungrib
+#-----------------------------------------------------------------------
+# Specify Execution Areas
+#-----------------------------------------------------------------------
 #
+export HOMErrfs=${HOMErrfs} #comes from the workflow at runtime
+export EXECrrfs=${EXECrrfs:-${HOMErrfs}/exec}
+export FIXrrfs=${FIXrrfs:-${HOMErrfs}/fix}
+export PARMrrfs=${PARMrrfs:-${HOMErrfs}/parm}
+export USHrrfs=${USHrrfs:-${HOMErrfs}/ush}
```

Comment:
For reference, we will need to explicitly repeat lines 10-14 in each J-jobs. 
To confirm, is this our intention?

Reply:
Yes, this is intention.